### DISCUSSION TITLE
how to build the examples?

### DISCUSSION THREAD

**Discussion by ahsf** [chakra-ui/ark#2443] [General]
In `frameworks/*/src/components/*/examples` are a lot of useful examples.
Is there a way to build them?
Also: is there some kind of sample CSS (while I very much like the headless approach, it would be really helpful to get started to have some sample CSS).

If there is no way to build the examples: would you accept a PR?

**very** happy with ark so far!

**Comment by cschroeter** [comment]
@ahsf 

All examples are used in the docs but also in the storybooks. For example you can run `bun react storybook`, `bun vue storybook` or `bun solid storybook` to see the examples in action.

There are some basic styles available in `./storybook/` folder or you can checkout park-ui.com for a more sophisticated example

**Reply by ahsf** [reply]
thanks, got it.
The CSS is actually perfect. Good enough to see whats going on, definitely ugly enough that it can't go into production.
